packages/marketing-analytics-browser/src/typings/browser-snippet.d.ts

```diff
@@ -4,4 +4,7 @@ declare global {
   // globalThis only includes `var` declarations
   // eslint-disable-next-line no-var
   var amplitude: InstanceProxy & { invoked: boolean };
+
+  // eslint-disable-next-line no-var
+  var amplitudeGTM: InstanceProxy & { invoked: boolean };
```

Comment:
Will this show for all MA users? Can we only include this in GTM?

Reply:
I think this is only dev types, and not exposed in prod